drivers/wireless/ieee80211/bcm43xxx/bcmf_netdev.h

```diff
@@ -35,6 +35,7 @@ int bcmf_netdev_register(FAR struct bcmf_dev_s *priv);
 
 void bcmf_netdev_notify_rx(FAR struct bcmf_dev_s *priv);
 
-void bcmf_netdev_notify_tx_done(FAR struct bcmf_dev_s *priv);
+void bcmf_netdev_notify_tx_done(FAR struct bcmf_dev_s *priv,
+                                clock_t delayms);
```

Comment:
don't need timeout here, since bcmf_sdpcm_sendframe and bcmf_receive can guarantee to restart the transmit ASAP once some memory is free.